bindings/python/eblob_python.cpp

```diff
@@ -72,13 +96,18 @@ struct eblob_py_iterator : eblob_iterate_control, boost::python::wrapper<eblob_i
 	}
 
 	static int iterator(struct eblob_disk_control *dc, struct eblob_ram_control *rc __attribute__((unused)),
-			void *data, void *priv, void *thread_priv __attribute__((unused)))
+			    int fd, uint64_t data_offset, void *priv, void *thread_priv __attribute__((unused)))
 	{
+		std::auto_ptr<char> p(new char[dc->data_size]);
```

Comment:
This binding is not written using newest C++11 standard. Also auto_ptr is convenient and safe (it plays simple role here, without auto_ptr assignments or using it as a function argument) here.